### PR TITLE
P4RT-5.1 traceroute packetin fix

### PR DESCRIPTION
**Pull request by singh-prem**
Fixed issues in P4RT 5.1 test:
1. Traceroute TTL=1 traffic being sent was "for-us ip", it has been modified to ATE port1  to ATE port2
2. Added `Priority: 1` to acl entries matching on TTL
3. A flag for ipv6 programming table entry was being used to delete entry instead of create, have fixed it.
4. Have removed TTL=0 case, as its not a need anymore.
5. Have enabled strict check for metadata contents.

### CODE REVIEW

File: feature/experimental/p4rt/ate_tests/traceroute_packetin_test/packetin_test.go [hunk 1]
```diff
@@ -163,92 +163,67 @@ func testPacketIn(ctx context.Context, t *testing.T, args *testArgs, IsIpv4 bool
...
-		TTL  uint8
-	}{{
-		desc: "TTL/HopLimit 1",
-		TTL:  1,
-	}, {
-		desc: "TTL/HopLimit 0",
-		TTL:  0,
-	}}
+	t.Log("TTL/HopLimit 1")
+	testTraffic(t, args.ate, args.packetIO.GetTrafficFlow(args.ate, IsIpv4, 1 /*ttl/Hoplimit=1*/, 300, 2), srcEndPoint, 10)
```
Comment (thesrinath) [inline feature/experimental/p4rt/ate_tests/traceroute_packetin_test/packetin_test.go:167]: Is this leftover stray comment?

**Review by vosipchu — APPROVED** [review]

**Comment by rsindhujha** [comment]
hi @singh-prem ,
The WBB testplan says client should receive packetin traffic with ttl=0 and ttl1 or ttl2. What's the reason to remove it?

**Comment by vosipchu** [comment]
Hi @rsindhujha , the HLD is updated right now to have just TTL=1 (this is what we expect). The updated version might have not been sent out yet, but it is there. Feel free to refer to me if there are questions, I was the source of this update.

File: feature/experimental/p4rt/ate_tests/traceroute_packetin_test/packetin_test.go (hunk 1 above)
Comment (vbhan-cisco) [reply feature/experimental/p4rt/ate_tests/traceroute_packetin_test/packetin_test.go:167]: > Is this leftover stray comment?

Yes, will remove it

Comment (singh-prem) [reply feature/experimental/p4rt/ate_tests/traceroute_packetin_test/packetin_test.go]: done

**Comment by xw-g** [comment]
Thanks for the contribution. The suggestion looks reasonable.

Going forward please help try to do one PR one thing. It would help increase the review velocity and quality. For example, this PR could have been split into 5 small and quick PRs each clearly covering just one topic. Thanks :)

**Comment by singh-prem** [comment]
> Thanks for the contribution. The suggestion looks reasonable.
> 
> Going forward please help try to do one PR one thing. It would help increase the review velocity and quality. For example, this PR could have been split into 5 small and quick PRs each clearly covering just one topic. Thanks :)

Thanks @xw-g , will follow the suggestion. For this particular PR, I would not have been able to successfully run the test with individual changes.

**Review by thesrinath — APPROVED** [review]
The got/want is reversed. Please update in a follow up PR

**Pull request merged**